src/settings/StreamUriResolverFromFile.h

```diff
@@ -3,7 +3,7 @@
 
 struct StreamUriResolverFromFile : winrt::implements<StreamUriResolverFromFile, winrt::Windows::Web::IUriToStreamResolver>
 {
-    WCHAR base_path[MAX_PATH];
+    WCHAR base_path[MAX_PATH] = L"";
```

Comment:
This can have it's own PR.

Reply:
opened https://github.com/microsoft/PowerToys/pull/5928